hw/ip/pattgen/data/pattgen_testplan.hjson

```diff
@@ -107,25 +107,39 @@
     {
       name: pattgen_op_cg
       desc: '''
-            The following coverage strategy is implemented:
-            * The counters are covered at the max
-              and min corner cases and for the middle values
-            * Data bus has functional coverage using walking ones and zeroes
-            * One bit width variables are wrapped in one cover point
-              while counters and data have designated cover points
-            * Channel enable signal is stand alone cover point
+            Covers the ch0 pattern generation complete interrupt and event done.
+            Covers the ch1 pattern generation complete interrupt and event done.
```

Comment:
Okay, I was actually confused why we collect both interrupt and event done, they seem to be very similar in design. https://github.com/lowRISC/opentitan/blob/master/hw/ip/pattgen/rtl/pattgen_core.sv#L68

Reply:
Whenever there is a event, interrupt is triggered

Reply:
I am not sure what was the intention of having two coverpoints. It was implemented by the previous owner.

Reply:
Okay, sounds good then. I will document this and check with Martin to see if it is needed.